setup.cfg

```diff
@@ -1,43 +0,0 @@
-[metadata]
-name = pandas-stubs
```

Comment:
I haven't tried your process yet locally, but I don't think that the metadata listed here is getting into the wheel when poetry is used to build the wheel.  Can you look into that?

Reply:
I think it is the second line from pyproject.toml, poetry handles that name to put into the wheel.

Reply:
It's not just that line, it's all the other lines in the metadata section of `setup.cfg`